Dockerfile.dev

```diff
@@ -28,7 +29,7 @@ RUN echo '{ "allow_root": true }' > /root/.bowerrc && \
     apk del .gyp python make g++ git && \
     bower cache clean && \
     npm uninstall -g bower && \
-    npm cache clean
+    npm cache clean --force
```

Comment:
why?

Reply:
https://docs.npmjs.com/cli/cache#details

npm stores cache data in an opaque directory within the configured cache, named _cacache. This directory is a cacache-based content-addressable cache that stores all http request data as well as other package-related data. This directory is primarily accessed through pacote, the library responsible for all package fetching as of npm@5.

All data that passes through the cache is fully verified for integrity on both insertion and extraction. Cache corruption will either trigger an error, or signal to pacote that the data must be refetched, which it will do automatically. For this reason, it should never be necessary to clear the cache for any reason other than reclaiming disk space, thus why clean now requires --force to run.

There is currently no method exposed through npm to inspect or directly manage the contents of this cache. In order to access it, cacache must be used directly.

npm will not remove data by itself: the cache will grow as new packages are installed.